contracts/modules/SSVDAO.sol

```diff
@@ -3,25 +3,22 @@ pragma solidity 0.8.18;
 
 import "../interfaces/ISSVDAO.sol";
 import "../libraries/Types.sol";
-import "../libraries/OperatorLib.sol";
 import "../libraries/ProtocolLib.sol";
-import "../libraries/CoreLib.sol";
-import "../libraries/SSVStorage.sol";
 
 contract SSVDAO is ISSVDAO {
     using Types64 for uint64;
     using Types256 for uint256;
 
     using ProtocolLib for StorageProtocol;
-    
+
     uint64 private constant MINIMAL_LIQUIDATION_THRESHOLD = 100_800;
 
     function updateNetworkFee(uint256 fee) external override {
-        uint64 previousFee = SSVStorageProtocol.load().networkFee;
-
-        SSVStorageProtocol.load().updateNetworkFee(fee);
+        StorageProtocol storage sp = SSVStorageProtocol.load();
+        uint64 previousFee = sp.networkFee;
 
-        emit NetworkFeeUpdated(previousFee, fee);
+        sp.updateNetworkFee(fee);
+        emit NetworkFeeUpdated(previousFee.expand(), fee);
```

Comment:
Make sure we have good unit test for this as we should not miss any expand / shrinks anywhere

I dont see any test edited to accompany this change which seems like we dont have a test for it.

Reply:
Thanks, Andrew. I pushed the test. It confirms that the previous version didn't emit the right `previousFee` value.